mobile/components/theme/index.js

```diff
@@ -4,7 +4,7 @@ export default {
     default: '#FFFFFF',
     reverse: '#16171A',
     wash: '#F5F8FC',
-    border: '#DFE7EF',
+    border: '#A3AFBF',
```

Comment:
Hold up why are we switching this around? Should we also switch this around in the web theme?

Reply:
The reason for this is that on mobile we can use `StyleSheet.hairlineWidth`, which renders the thinnest possible line for that display; on a retina display (basically anything made in the last 5+ years), that thin line needs extra contrast, so I darkened it on mobile. That being said, I understand why that's a bad and confusing distinction - I'm going to just add a 'hairline' color on mobile theme.